xml/admin_ds_backup.xml

```diff
@@ -140,5 +142,92 @@
     </para>
    </listitem>
   </itemizedlist>
+ </sect2>
+ </sect1>
+ <sect1 xml:id="restore-ceph">
+  <title>Restoring a Ceph Node</title>
+  <para>
+   The basic procedure for recovering a node from backup is to reinstall the
+   node, replace its configuration files, and then re-orchestrate the cluster
+   so that the replacement node is re-added. 
+  </para>
+  <orderedlist>
+   <listitem>
+    <para>
+     Re-install the node. (See <link
+     xlink:href="https://documentation.suse.com/ses/6/single-html/ses-deployment/#ceph-install-stack"/>
+     steps 1-4.)
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Update to the latest patch level (See: <link
+      xlink:href="https://documentation.suse.com/sles/15-SP1/single-html/SLES-admin/#sec-zypper-softup-update"/>.)
```

Comment:
Ditto, please use an xref for anything internal to SES books.

Reply:
But this is *not* a SES book, it is the SLE Admin Guide.

Reply:
So this one is. Ignore that ;)

Reply:
TL;DR - SES = yes. SLE = no.